prefix.go

```diff
@@ -336,7 +344,10 @@ func (i *ipamer) AcquireSpecificIP(ctx context.Context, prefixCidr, specificIP s
 // If there is no free IP an NoIPAvailableError is returned.
 // If the Prefix is not found an NotFoundError is returned.
 func (i *ipamer) acquireSpecificIPInternal(ctx context.Context, namespace, prefixCidr, specificIP string) (*IP, error) {
-	prefix := i.PrefixFrom(ctx, prefixCidr)
+	prefix, err := i.PrefixFrom(ctx, prefixCidr)
+	if err != nil {
+		return nil, fmt.Errorf("%w: unable to find prefix for cidr:%s error:%s", ErrNotFound, prefixCidr, err.Error())
```

Comment:
No need to wrap this error again.

Reply:
not done, otherwise checking for the error is backend dependend